packages/supabase_flutter/lib/src/supabase_auth.dart

```diff
@@ -159,6 +159,7 @@ class SupabaseAuth with WidgetsBindingObserver {
 
   @override
   void didChangeAppLifecycleState(AppLifecycleState state) {
+    // ignore:missing_enum_constant_in_switch
```

Comment:
There was a new `AppLifecycleState.hidden` enum value added in the latest stable version of Flutter. This line was added to ignore the check to add all enum values to the switch.